src/test/functional/cypress.config.js

```diff
@@ -0,0 +1,176 @@
+const {defineConfig} = require('cypress');
+
+/// <reference types="cypress" />
+const axios = require('axios');
+
+module.exports = defineConfig({
+  defaultCommandTimeout: 6000,
+  video: false,
+  scrollBehavior: 'top',
+  experimentalInteractiveRunEvents: true,
```

Comment:
check whether this feature still experimental

Reply:
yes

https://docs.cypress.io/guides/references/experiments#Configuration